postgres-appliance/scripts/configure_spilo.py

```diff
@@ -251,7 +251,12 @@ def deep_update(a, b):
     {{/PAM_OAUTH2}}
     - host    all             all                ::1/128            md5
     - hostssl replication     {{PGUSER_STANDBY}} all                md5
+    {{#ALLOW_NOSSL}}
```

Comment:
Unfortunately it is still wrong.

This block should be after(instead) of https://github.com/zalando/spilo/pull/366/files#diff-5e29afdf8025daafbc08cc254eb83f9fR263. The issue is that [PAM](https://github.com/zalando/spilo/pull/366/files#diff-5e29afdf8025daafbc08cc254eb83f9fR261) requires the password transmitted as it is, therefore connection must be encrypted.

The change https://github.com/zalando/spilo/pull/366/files#diff-5e29afdf8025daafbc08cc254eb83f9fR257-R259 is fine.

Reply:
ok, I've read the docs on pg_hba.conf and i think i understand what you're saying. sorry for all the back and forth on this!